moont/README.md

```diff
@@ -0,0 +1,20 @@
+# MoOnt
+
+This [W3ID](https://w3id.org) provides a persistent URI namespace for the MoOnt ontology.
+
+## Uses
+
+The MoOnt ontology is a graph data structure that conceptualizes the domain of the Modelica simulation language.
+
+## Contact
+
+This space is administered by:
+
+**Elisabeth Eckstädt**
+_R&D Engineer_
+[Fraunhofer IIS/EAS](hhttps://www.eas.iis.fraunhofer.de/)
+Dresden, Germany
+<elisabeth.eckstaedt@eas.iis.fraunhofer.de>
+GitHub: [ElisEck](https://github.com/ElisEck)
+Linkedin: [Elisabeth Eckstädt](https://www.linkedin.com/in/elisabeth-eckst%C3%A4dt-18689115a)
+ORCID: [0000-0003-4396-5307](https://orcid.org/0000-0003-4396-5307)
```

Comment:
To preserve desired line-breaks...
```suggestion
**Elisabeth Eckstädt**  
_R&D Engineer_  
[Fraunhofer IIS/EAS](hhttps://www.eas.iis.fraunhofer.de/)  
Dresden, Germany  
<elisabeth.eckstaedt@eas.iis.fraunhofer.de>  
GitHub: [ElisEck](https://github.com/ElisEck)  
Linkedin: [Elisabeth Eckstädt](https://www.linkedin.com/in/elisabeth-eckst%C3%A4dt-18689115a)  
ORCID: [0000-0003-4396-5307](https://orcid.org/0000-0003-4396-5307)
```

Reply:
This is happening on many PRs.  The trailing spaces seems a bit magic.  Maybe better to suggest bulleted lists?

Reply:
@davidlehn — Bullet lists aren't really appropriate for this content. Trailing double (or more) spaces are indeed Markdown magic. The other option is HTML `<br>` tags.